src/repo/util.js

```diff
@@ -223,9 +223,92 @@ const trimRecords = async (recordList, { history = false, user = null } = {}) =>
     return result;
 };
 
+/**
+ * Given an EvidenceLevel's displayName, returns the corresponding normalized EvidenceLevel
+ * Based on Cam Grisdale' script (https://www.bcgsc.ca/jira/browse/GERO-289)
+ *
+ * @param {String} displayName the EvidenceLevel's displayName
+ * @returns {String} the normalized PORI EvidenceLevel
+ *
+ * @example
+ * > normalizeEvidenceLevel('CIViC B3')
+ * 'B'
+ */
+const normalizeEvidenceLevel = (displayName) => {
```

Comment:
Are the normalized evidence levels based on a particular definition? Either way it would probably make more sense to add them to GraphKB itself with edges and get the normalization from there rather than hard-coding anything